package.json

```diff
@@ -33,16 +35,23 @@
     "url": "https://github.com/Shopify/tslint-config-shopify/issues"
   },
   "devDependencies": {
+    "@types/chalk": "^0.4.31",
+    "@types/jsdom": "^11.0.0",
     "@types/node": "^7.0.5",
+    "chalk": "^1.1.3",
+    "isomorphic-fetch": "^2.2.1",
+    "jsdom": "^11.0.0",
     "npm-run-all": "^4.0.2",
     "rimraf": "^2.6.0",
+    "ts-node": "^3.0.6",
     "tslint": "^5.1.0",
     "typescript": "^2.3.2"
   },
   "peerDependencies": {
     "tslint": "^5.1.0"
   },
   "dependencies": {
+    "node-emoji": "^1.5.1",
```

Comment:
Presumably you still need the `@types` for this, just put it in dev dependencies instead of regular dependencies